tests/base/src/test/java/org/wildfly/security/auth/client/CredentialStoreSaslAuthenticationTest.java

```diff
@@ -19,6 +19,7 @@
 package org.wildfly.security.auth.client;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.*;
```

Comment:
It's good practice to avoid `.*` imports and just import specifically what is needed.